confidential-data-hub/kms/src/error.rs

```diff
@@ -3,7 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-use attestation_agent::aa_kbc_params;
+use attestation_agent::config::aa_kbc_params;
```

Comment:
This PR also moves `aa_kbc_params` to another place. Could we bring this part out as a separate commit?

Reply:
I think these two parts can be classified as a commitment, because aa_ Kbc_ Parameters themselves are part of the AA configuration parameters, so I think it is OK to put them in a commit.